CHANGELOG.md

```diff
@@ -177,6 +177,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Interpret byte array as primitive using VarHandles ([#11362](https://github.com/opensearch-project/OpenSearch/pull/11362))
 - Automatically add scheme to discovery.ec2.endpoint ([#11512](https://github.com/opensearch-project/OpenSearch/pull/11512))
 - Restore support for Java 8 for RestClient ([#11562](https://github.com/opensearch-project/OpenSearch/pull/11562))
+- Modified _cat/shards to include deleted docs count in the response  ([#11678](https://github.com/opensearch-project/OpenSearch/pull/11678))
```

Comment:
Nitpick, but let's match the tone by using the imperative, i.e. "Modify" versus "Modified". Alternatively you can use your PR title/commit message "Add deleted doc count in _cat/shards" as that is great too!